packages/menu/src/Menu/Menu.js

```diff
@@ -0,0 +1,75 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { themeGet } from 'styled-system'
+import { Button } from 'pcln-design-system'
+import { ChevronDown } from 'pcln-icons'
+import Popover from 'pcln-popover'
+import MenuList from '../MenuList'
+
+const LinkButton = styled(Button)`
+  display: flex;
+  align-items: center;
+  padding: ${themeGet('space.2')}px;
+`
+
+function Menu({ id, buttonText, color, height, size, children, ...props }) {
+  const MenuContent = ({ handleClose }) => (
+    <MenuList
+      id={id}
+      color={color}
+      size={size}
+      height={height}
+      handleClose={handleClose}
+    >
+      {children}
+    </MenuList>
+  )
+
+  MenuContent.propTypes = {
+    handleClose: PropTypes.func,
+  }
+
+  const onKeyDown = useCallback((evt) => {
+    const key = evt.key
+    if (key === 'ArrowDown' || key === 'ArrowUp') {
+      evt.preventDefault()
+      evt.target.click()
```

Comment:
can't you just call the handler here? firing a click on keypress feels wrong.

Reply:
I tried to use useState and isOpen but it breaking some of the default functionality like the escape key closing or clicking on the background. This might be a bug with the Popover, I'm going to leave it how it is for now and add an issue to look into this